diagnostic.md

```diff
@@ -3,36 +3,51 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+- give client-side application a lot of structure right out of the box
+- give app a lot of front-end best practices, standards, and conventions
+- build a state-dependent UI in a proven, standardized way
```

Comment:
👍 
Some other reasons might be:
Front end routing
Easily display dynamic content using two-way binding
No more AJAX - Ember takes care of making API calls for you.
1/1